### PR TITLE
Update oidc with info on Custom Error Redirect

### PR DESCRIPTION
**Pull request by sc-roblloyd**
Changes to support fix for next platform release 1.0.257

### CODE REVIEW

**Comment by robertsoniv** [comment]
@sc-roblloyd I will wait to merge until the day of the platform release (reason for this is because the API has to be released for the API reference to regenerate via the production Open API spec). In the future, mark your PR as a draft if it's related to an upcoming platform release so that it doesn't accidentally get merged too soon.

**Comment by sc-roblloyd** [comment]
@robertsoniv - Thanks! my bad, I will mark as draft from now on for future platform release.

File: content/documents/sso-via-openid-connect.mdx [hunk 1]
```diff
@@ -219,4 +220,18 @@ Now once login via OpenID is successful OrderCloud will redirect your user to th
...
+
+If an error occurs, OrderCloud will redirect the user to that custom error url.
+
+For example:
+
+```
+https://my-buyer-application.com/error?ErrorMessage={0}
+
+```
+The error message `ErrorMessage={0}` will be replaced with the error text from OrderCloud.
```
Comment (crhistianramirez) [inline content/documents/sso-via-openid-connect.mdx:237]: Did you mean to write that `{0}` will be replaced with the error text from OrderCloud?

Comment (sc-roblloyd) [reply content/documents/sso-via-openid-connect.mdx]: Yes, corrected it. 

@crhistianramirez - please let me know if you have any other suggestions.

File: content/documents/sso-via-openid-connect.mdx [hunk 2]
```diff
@@ -88,7 +88,8 @@ Content-Type: application/json
     "AppStartUrl": "https://my-buyer-application.com/login?token={0}",
     "AuthorizationEndpoint": "…",
     "TokenEndpoint": "…",
-    "IntegrationEventID": "integrationeventID" // required for configs created after 03/31/2020
+    "IntegrationEventID": "integrationeventID", // required for configs created after 03/31/2020
+    "CustomErrorUrl": "https://my-buyer-application.com/error?ErrorMessage={0}" // optional redirect to direct to a custom url upon error
```
Comment (crhistianramirez) [inline content/documents/sso-via-openid-connect.mdx:92]: Seems a bit wordy, how do you feel about this?

> optionally redirect to a custom URL upon error

**Review by robertsoniv — COMMENTED** [review]
@sc-roblloyd I'm gonna merge this since 1.0.257 is getting released today. Think you can update the "updatedDate" to today? Also, @crhistianramirez did Rob address all of your concerns with is latest commits?

**Comment by sc-roblloyd** [comment]
@robertsoniv - date has been updated

**Review by robertsoniv — APPROVED** [review]
